hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/RepeatableHapiTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.junit;
+
+import static com.hedera.services.bdd.junit.TestTags.ONLY_REPEATABLE;
+import static org.junit.jupiter.api.parallel.ResourceAccessMode.READ;
+
+import com.hedera.services.bdd.junit.extensions.NetworkTargetingExtension;
+import com.hedera.services.bdd.junit.extensions.SpecNamingExtension;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.ResourceLock;
+
+/**
+ * Annotation for a {@link HapiTest} that can only be run in repeatable mode. The {@link EmbeddedReason} annotation
```

Comment:
`{@link EmbeddedReason}` -> `{@link RepeatableReason}`, and on next line isn't it `embedded` -> `repeatable` too?

Reply:
Ah yes will fix this typo in future PR since Determinism checks are being flaky today. 😝